ui/Screen/SendFunds.svelte

```diff
@@ -0,0 +1,256 @@
+<script>
+  import { getContext } from "svelte";
+  import { fromStore, toStore, amountStore } from "../src/transfer.ts";
+
+  import { Dropdown, ModalLayout, Rad } from "../DesignSystem/Component";
+  import Row from "../DesignSystem/Component/Transaction/Row.svelte";
+  import {
+    Avatar,
+    Button,
+    Icon,
+    Input,
+    Text,
+    Title,
+  } from "../DesignSystem/Primitive";
+
+  const { identity } = getContext("session");
+  const { orgs } = getContext("session");
+
+  let fromAddress;
+  fromStore.subscribe(value => {
+    fromAddress = value;
+  });
+  let toAddress;
+  toStore.subscribe(value => {
+    toAddress = value;
+  });
+  let amount;
+  amountStore.subscribe(value => {
+    amount = value;
+  });
+
+  const dropdownOptions = [];
+  dropdownOptions.push({
+    variant: "avatar",
+    value: identity.accountId,
+    avatarProps: {
+      variant: "circle",
+      title: identity.metadata.handle,
+      avatarFallback: identity.avatarFallback,
+      imageUrl: identity.imageUrl,
+    },
+  });
+  for (let i = 0; i < orgs.length; i++) {
+    dropdownOptions.push({
+      variant: "avatar",
+      value: orgs[i].accountId,
+      avatarProps: {
+        variant: "square",
+        title: orgs[i].id,
+        avatarFallback: orgs[i].avatarFallback,
+        imageUrl: null,
+      },
+    });
+  }
+
+  let step = 1;
+
+  const nextStep = () => {
```

Comment:
Instead of using magic steps, we can use enumerated states so it's easier to reason about:
```
step === TransactionSubmission
```` 

means more than 
```
step === 1
```

Check out some of the other flows like https://github.com/radicle-dev/radicle-upstream/blob/master/ui/Screen/OrgRegistration.svelte for examples

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/712/commits/e2d6bce27f3b132de219c3eaf121c97744e9bbac